core/trace/src/main/scala/org/typelevel/otel4s/trace/SpanContext.scala

```diff
@@ -67,12 +80,30 @@ object SpanContext {
     val Bytes: Int = 16
     val HexLength: Int = Bytes * 2
     val InvalidHex: String = "0" * HexLength
+    val Invalid: ByteVector = ByteVector.fromValidHex(InvalidHex)
```

Comment:
this line is objectively hilarious. no comments; 10/10

Reply:
lol, I didn't even notice this at first